src/pages/index.js

```diff
@@ -0,0 +1,266 @@
+import "./index.css";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+import { setButtonText, deleteButtonText } from "../utils/helpers.js";
+
+const editModalBtn = document.querySelector(".profile__edit-button");
+const cardModalBtn = document.querySelector(".profile__new-post-button");
+const profileName = document.querySelector(`.profile__name-main`);
+const profileDescription = document.querySelector(`.profile__name-description`);
+
+const profileAvatar = document.querySelector(`.profile__avatar`);
+const avatarModalBtn = document.querySelector(`.profile__avatar-btn`);
+
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarModalCloseBtn = avatarModal.querySelector(".modal__close-btn");
+const avatarFormElement = avatarModal.querySelector(`.modal__form`);
+const avatarModalLinkInput = avatarModal.querySelector(`#profile-avatar-input`);
+
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = deleteModal.querySelector(".modal__form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms